src/TextField/TextFieldLabel.js

```diff
@@ -35,6 +55,11 @@ const propTypes = {
    * Override the inline-styles of the root element.
    */
   style: PropTypes.object,
+
```

Comment:
It seems like eslint might be missing this style of declaring PropTypes. These aren't in order:

disabled
shrink
htmlFor
onTouchTap
style
shrinkStyle